docs/Installation/Installation_Guide.rst

```diff
@@ -172,6 +172,54 @@ configuring persistent voloumes at 500 Gi/node.
 
    It is a best practice to install Zenko on a fresh cluster.
 
+Pod Disruption Budgets
+**********************
+
+Zenko relies on several stateful services that require a minimum number
+of pods to function with high availability, with resilence under many
+outage scenarios. Pod disruption budgets set how many pods of a given
+application can safely fail and continue to operate normally before
+Kubernetes disables access to the service.
+
+.. note::
+
+   If you are installing Zenko in a high-availability production environment,
+   set a pod disruption budget. If you are installing a basic Zenko deployment
+   (for testing or familiarization, for instance) you can skip this step.
+
+The following installed applications allow for configuring disruption
+budgets:
+
+- MongoDB
+- Redis
+- Zenko-Quorum
+
+In a three-node cluster, Zenko configures a ``maxUnavailable`` budget
+of 1 by default. However in larger clusters, this should be increased to
```

Comment:
Is the default set by Zenko always 1, regardless of the number of nodes? Could it make sense to set it to "nodes / 2" (integral division) by default, so that it maps better to the number of nodes?

Reply:
IMHO, we shouldn't rely too much on `nodeCount` for this sort of things.

Reply:
Yeah I agree with @giacomoguiulfo I think it would be best to not to rely on templating 'magic' to get to an appropriate number here based off node count and would be best as a conscious decision to be made by the installer. It would likely also remove the ability to specify less if necessary (if someone still only wants 1 failure in a 5+ nodes configuration).